src/main/java/com/google/sps/Data/Request.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+
+public final class Request {
```

Comment:
It is better to rename it to be something like `RequestUtil` or `RequestUtils`. Generic names like `Request`, or `Response` will be more likely used as interfaces.